plugin.video.cbc/addon.xml

```diff
@@ -1,27 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.cbc"
        name="Canadian Broadcasting Corp (CBC)"
-       version="3.0.3"
-       provider-name="t1m">
+       version="4.0.0"
+       provider-name="micahg">
```

Comment:
I don't think this is fair, I'd prefer if you add yourself as an author along @learningit. But it is really up to him.

Reply:
I have no issues at all with this. I suggested that he take over this I'd as I no longer maintain the add-on.

Reply:
ID not I'd in above

Reply:
ID not I'd in above

Reply:
fine to me then

Reply:
Actually, as soon as I submitted this I thought its probably too much also. What is the syntax for sharing - comma-delimiting?

Reply:
NVM, just read https://kodi.wiki/view/Addon.xml#provider-name_attribute